### PR TITLE
Add VIA support for kbd67rev1

### PR DESCRIPTION
**Pull request by daisukekoba**
## Description

Add VIA support for kbdfans/kbd67/rev1

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd67/rev1/keymaps/via/config.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2018 MechMerlin
```
Comment (zvecr) [inline keyboards/kbdfans/kbd67/rev1/keymaps/via/config.h:1]: Can we delete this file please

File: keyboards/kbdfans/kbd67/rev1/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+   * |Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift | Up|End |
+   * |----------------------------------------------------------------|
+   * |Ctrl|Win |Alt |        Space          |Alt| FN|Ctrl|Lef|Dow|Rig |
+   * `----------------------------------------------------------------'
+   */
+[0] = LAYOUT_65_ansi(
+  KC_ESC,  KC_1,    KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_HOME, \
+  KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP, \
+  KC_LCTL, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT, KC_PGDN,  \
+  KC_LSFT, KC_Z,    KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,           \
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/rev1/keymaps/via/keymap.c:36]:
```suggestion
  KC_ESC,  KC_1,    KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_HOME,
  KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
  KC_LCTL, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT, KC_PGDN,
  KC_LSFT, KC_Z,    KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
```

File: keyboards/kbdfans/kbd67/rev1/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+   * |        |   |   |   |   |   |   |   |End|PDn|Dow|      |PUp|    |
+   * |----------------------------------------------------------------|
+   * |    |    |    |                       |   |   |    |Hom|PDn|End |
+   * `----------------------------------------------------------------'
+   */
+[1] = LAYOUT_65_ansi(
+   KC_GRV,  KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9, KC_F10, KC_F11, KC_F12,          KC_DEL,KC_INS, \
+   KC_CAPS,_______,  KC_UP,_______,_______,_______,_______,_______,KC_PSCR,KC_SLCK,KC_PAUS,  KC_UP,_______,     _______,_______,   \
+      _______,KC_LEFT,KC_DOWN,KC_RGHT,_______,_______,_______,_______,KC_HOME,KC_PGUP,KC_LEFT,KC_RGHT,            _______,_______, \
+           _______,_______,_______,_______,_______,_______,_______,_______, KC_END,KC_PGDN,KC_DOWN,       _______,KC_PGUP,_______, \
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/rev1/keymaps/via/keymap.c:56]:
```suggestion
   KC_GRV,  KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9, KC_F10, KC_F11, KC_F12,          KC_DEL,KC_INS,
   KC_CAPS,_______,  KC_UP,_______,_______,_______,_______,_______,KC_PSCR,KC_SLCK,KC_PAUS,  KC_UP,_______,     _______,_______,
      _______,KC_LEFT,KC_DOWN,KC_RGHT,_______,_______,_______,_______,KC_HOME,KC_PGUP,KC_LEFT,KC_RGHT,            _______,_______,
           _______,_______,_______,_______,_______,_______,_______,_______, KC_END,KC_PGDN,KC_DOWN,       _______,KC_PGUP,_______,
```

File: keyboards/kbdfans/kbd67/rev1/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    _______,  _______,  _______,                     _______,                     _______,_______,_______,KC_HOME,KC_PGDN, KC_END),
+
+
+[2] = LAYOUT_65_ansi(
+  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,        _______,_______, \
+   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,     _______,_______,   \
+      _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,            _______,_______, \
+           _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,       _______,_______,_______, \
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/rev1/keymaps/via/keymap.c:64]:
```suggestion
  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,        _______,_______,
   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,     _______,_______,
      _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,            _______,_______,
           _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,       _______,_______,_______,
```

File: keyboards/kbdfans/kbd67/rev1/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    _______,  _______,  _______,                     _______,                     _______,_______,_______,_______,_______,_______),
+
+
+[3] = LAYOUT_65_ansi(
+  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,        _______,_______, \
+   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,     _______,_______,   \
+      _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,            _______,_______, \
+           _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,       _______,_______,_______, \
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/rev1/keymaps/via/keymap.c:72]:
```suggestion
  _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,        _______,_______,
   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,     _______,_______,
      _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,            _______,_______,
           _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,       _______,_______,_______,
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**